Backpack/src/main/java/net/skyscanner/backpack/skeleton/BpkImageSkeleton.kt

```diff
@@ -55,7 +55,7 @@ class BpkImageSkeleton @JvmOverloads constructor(
 
   init {
     backgroundDrawable = AppCompatResources.getDrawable(context, R.drawable.bpk_skeleton_bg) as GradientDrawable
-    background = backgroundDrawable
+    background = backgroundDrawable.mutate()
```

Comment:
Call mutate() to get a new background drawable to avoid affecting other skeleton views.

Reply:
ah thank you - good catch!